### PR TITLE
generate code without runtime dependency on DLL (Fixes #58)

### LINKED ISSUE #58: Send Ctrl+C does not work on Windows 8.1
Windows 8.1 Pro

When running manually:
image

When running via Java API:
```
Exception in thread "main" org.jvnet.winp.WinpException: External Ctrl+C execution failed for process pid=1160. Ctrl+C process exited with code -1073741515: Failed to attach to the console (see the AttachConsole WinAPI call). error=0 at winp.cpp:59
	at org.jvnet.winp.Native.sendCtrlC(Native Method)
	at org.jvnet.winp.Native.sendCtrlC(Native.java:90)
	at org.jvnet.winp.WinProcess.sendCtrlC(WinProcess.java:93)
	at example.Main.main(Main.java:29)
```
See example project to reproduce: [stop-child-process.zip](https://github.com/kohsuke/winp/files/2571422/stop-child-process.zip).

### PR DESCRIPTION
**Pull request by segrey**
Additionally, update toolset to v140. Current toolset (v120) is not available for Visual Studio 17 and the only way is to install VC++ 2013 which is inconvenient. (https://social.msdn.microsoft.com/Forums/windows/en-US/07ce397a-611b-4233-99b7-4b4bf6f58589/is-there-a-way-to-install-platform-toolset-v120-on-visual-studio-2017)

### CODE REVIEW

**Comment by segrey** [comment]
This change can be split into two commits: updating toolset to v140 and adding `/MT` switch in Code Generation. @oleg-nenashev What do you think?

File: native/sendctrlc/sendctrlc.vcxproj [hunk 1]
```diff
@@ -142,6 +145,7 @@
       <IntrinsicFunctions>true</IntrinsicFunctions>
       <PreprocessorDefinitions>WIN32;NDEBUG;_CONSOLE;_LIB;%(PreprocessorDefinitions)</PreprocessorDefinitions>
       <SDLCheck>true</SDLCheck>
+      <RuntimeLibrary>MultiThreaded</RuntimeLibrary>
```
Comment (oleg-nenashev) [inline native/sendctrlc/sendctrlc.vcxproj]: this app is actually single-threaded IIRC, but should not make much difference. Will double check

Comment (segrey) [reply native/sendctrlc/sendctrlc.vcxproj]: By default "Runtime Library" is set to "Multi-threaded Debug DLL (/MDd)" for Debug and "Multi-threaded DLL (/MDd)".

Looks like there is no "single-threaded" option:
image

**Comment by segrey** [comment]
BTW, please note that binaries built with neither toolset v120 nor v140 will work on Windows XP (v140_xp is needed for that). Though, not sure it's needed at all.

**Comment by segrey** [comment]
@oleg-nenashev Any ETA for reviewing the PR?

**Comment by segrey** [comment]
@oleg-nenashev Thanks a lot!
>(looking for co-maintainers in general)

I can help you with that.

**Comment by oleg-nenashev** [comment]
@segrey Thanks! I will follow-up on that

**Comment by oleg-nenashev** [comment]
With the latest MVS 2017 the v141 toolkit is needed AFAICT. Tried upgrading in https://github.com/kohsuke/winp/pull/62 , but it seems that it is not going to fly until proper nddll.lib is passed to the build. So far I can pass the 8.1 only, but it won't produce the compatible result I'd guess

**Comment by segrey** [comment]
Ah, indeed, I have "VC++ 2015.3 v14.00 (v140) toolset for desktop" installed in "Tools | Get Tools and Features...".

**Comment by oleg-nenashev** [comment]
@segrey I believe I worked it around on the new laptop WinP 1.28 should become available in Maven Central in few hours, promotion is in progress

**Comment by segrey** [comment]
@oleg-nenashev Thanks! I've updated to WinP 1.28 and several related issues are fixed now!

**Comment by oleg-nenashev** [comment]
OK, thanks for the feedback! I should be able to cut releases regularly now

**Pull request merged**